cmake/templates/CharmConfig.cmake.in

```diff
@@ -1,3 +1,6 @@
+
+@PACKAGE_INIT@
```

Comment:
Could you add this command after **CHARM_PROJECT_VERSION** please? (see [this](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#example-generating-package-files))
Also the way, Cmake config package initializes the package version, it would be better to initialize **CHARM_PROJECT_VERSION** with **CMAKE_PROJECT_VERSION** instead.